brightwind/analyse/shear.py

```diff
@@ -273,14 +277,14 @@ def __init__(self, wspds, heights, min_speed=3, calc_method='power_law', by_mont
 
                     elif i == segments_per_day - 1:
                         start = str(start_times[i].time())
-                        end = str(start_times[0].time())
-                        time_wspds[i] = pd.DataFrame(anemometers_df).between_time(start, end, include_end=False)
+                        end = str((start_times[0] - resolution).time())
```

Comment:
@BiancaMorandi I wonder would there be a case where the resolution is incorrect?

Reply:
@stephenholleran yes, it could happen if timeseries has many data gaps but I thought that for shear it will be unlikely that this will happen (user will usually give as many data as possible and not very irregular data).  If we want to handle this then I need to add an option to have resolution as a user input to the function. 

It doesn't make much sense to me to have resolution as an input as this is only used to make sure that each segment of the day is not overlapping...it was done with `.between_time(start, end, include_end=False)` function without using `resolution`. This function will be updated to use different inputs and if I change the inputs to the latest version I cannot have compatibly with old versions of pandas.

Reply:
ok, thanks. Don't worry about it. As you said for shear the resolution should be well defined.